app/src/main/java/com/example/androidacademyhw/movies/FragmentMoviesList.kt

```diff
@@ -74,91 +77,13 @@ class FragmentMoviesList : Fragment() {
-                "Chris Evans",
-                R.drawable.actor2
-            )
-        )
-        actors.add(
-            Actor(
-                "Mark Ruffalo",
-                R.drawable.actor3
-            )
-        )
-        actors.add(
-            Actor(
-                "Chris Hemsworth",
-                R.drawable.actor4
-            )
-        )
-
-        movies.add(
-            Movie(
-                "Avengers: End Game",
-                13,
-                "Action, Adventure, Drama",
-                4,
-                125,
-                137,
-                R.drawable.smaller_movie,
-                "After the devastating events of Avengers: Infinity War, the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to reverse Thanos' actions and restore balance to the universe.",
-                false,
-                actors
-            )
-        )
-        movies.add(
-            Movie(
-                "Tenet",
-                16,
-                "Action, Sci-Fi, Thriller",
-                5,
-                98,
-                97,
-                R.drawable.tenet,
-                "After the devastating events of Avengers: Infinity War, the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to reverse Thanos' actions and restore balance to the universe.",
-                true,
-                actors
-            )
-        )
-        movies.add(
-            Movie(
-                "Black Widow",
-                13,
-                "Action, Adventure, Sci-Fi",
-                4,
-                38,
-                102,
-                R.drawable.black_widow,
-                "After the devastating events of Avengers: Infinity War, the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to reverse Thanos' actions and restore balance to the universe.",
-                false,
-                actors
-            )
-        )
-        movies.add(
-            Movie(
-                "Wonder Woman 1984",
-                13,
-                "Action, Adventure, Fantasy",
-                5,
-                74,
-                120,
-                R.drawable.wonderwomen,
-                "After the devastating events of Avengers: Infinity War, the universe is in ruins. With the help of remaining allies, the Avengers assemble once more in order to reverse Thanos' actions and restore balance to the universe.",
-                false,
-                actors
-            )
-        )
+    suspend fun getMovies(context: Context): List<Movie> {
+        return withContext(Dispatchers.IO) {
+            loadMovies(context)
+        }
     }
 
-    override fun onAttach(context: Context) {
+        override fun onAttach(context: Context) {
         super.onAttach(context)
         if (context is FragmentClickListener) {
             fragmentMoviesClickListener = context
```

Comment:
Does it work?

Reply:
@Heaven45 try to delete and see if everything keep s going well =)

Reply:
Deleted it. 
Everything works perfectly well without it.